stytra/stimulation/stimuli/generic_stimuli.py

```diff
@@ -111,7 +113,7 @@ def stop(self):
         """
         pass
 
-    def initialise_external(self, experiment):
+    def initialise_external(self, experiment, calibrator = -999):
```

Comment:
-999 is not a good default. I would put actually no default here, or `None` if needed for backwards compatibility.